tests/Unit/ApparentHorizons/StrahlkorperGrTestHelpers.cpp

```diff
@@ -59,6 +61,49 @@ tnsr::ii<DataType, SpatialDim, Frame> extrinsic_curvature_sphere(
   return extrinsic_curvature;
 }
 }  // namespace Minkowski
+
+namespace Kerr {
+template <typename DataType>
+Scalar<DataType> horizon_radius(const std::array<DataType, 2>& theta_phi,
+                                const double& mass,
+                                const std::array<double, 3>& spin) noexcept {
+  // Because magnitude() requires tensors, just compute
```

Comment:
do you mean magnitude_squared? Because clearly magnitude takes a `std::array`

Reply:
I forgot to delete this comment, which no longer applies after I changed the next line.